tools/src/main/java/org/apache/kafka/tools/ClientMetricsCommand.java

```diff
@@ -367,6 +368,16 @@ public void checkArgs() {
             if (has(alterOpt)) {
                 if ((isNamePresent && has(generateNameOpt)) || (!isNamePresent && !has(generateNameOpt)))
                     throw new IllegalArgumentException("One of --name or --generate-name must be specified with --alter.");
+
+                interval().ifPresent(intervalStr -> {
+                    if (!intervalStr.isEmpty()) {
+                        try {
+                            Integer.parseInt(intervalStr);
+                        } catch (NumberFormatException e) {
+                            throw new IllegalArgumentException("Invalid interval value. Must be a valid integer or empty to delete the setting.");
```

Comment:
Can this be better to specify?
```suggestion
                            throw new IllegalArgumentException("Invalid interval value. Enter an integer, or leave empty to delete.");
```

Reply:
Thanks for the suggestion!

Instead of vaguely stating that it should be a valid integer or empty, the new message clearly specifies that it must be an integer or left empty to delete.

Since empty values now reset the interval to its default value (as shown in the previous screenshot), I think we can explicitly say “reset” here. What do you think?